data_utils/pr_stats.py

```diff
@@ -62,45 +76,119 @@ def get_pr_stats(name, owner):
     return run_query(query)
 
 
-def process_stats_query_results(writer, result):
+def calculate_week(start_date, created_date):
+    """ Calculates the internship week.
+
+    Args:
+        start_date: intern start date in "mm/dd/YYYY" form.
+        created_date: pull request created date in ISO format.
+
+    Returns:
+        Internship week.
+    """
+    start_date = datetime.strptime(start_date, "%m/%d/%Y")
+    days = abs(created_date - start_date).days
+    return days // 7 + 1
```

Comment:
add some parens around days // 7; even if order of operations is on your side, it improves readability.